patches/chromium/revert_remove_contentrendererclient_shouldfork.patch

```diff
@@ -1,18 +1,26 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: deepak1556 <hop2deep@gmail.com>
-Date: Wed, 23 Oct 2019 15:30:18 -0700
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Tue, 4 Feb 2020 08:59:32 -0700
 Subject: Revert "Remove ContentRendererClient::ShouldFork."
 
-This reverts commit 6b068eb8ca4a3c7350bdafa22fc0cf0636ef8b74.
+This reverts the CL at https://chromium-review.googlesource.com/c/chromium/src/+/1812128.
+We use it to force a new renderer process for navigations, and need to start a new renderer process
+for every navigation to keep Node.js working properly. Once Native Modules in the renderer process
+are required to be NAPI or context aware (Electron v11), this patch can be removed.
```

Comment:
The changes from `content/renderer/render_frame_impl.cc` and `content/public/renderer/content_render_client.{cc|h}`  should also be reverted, it gives the hook for us to decide process restart.

Reply:
https://github.com/electron/electron/issues/20735 I haven't gotten to this yet.